packages/wrangler/src/pages/functions/buildPlugin.ts

```diff
@@ -8,12 +8,12 @@ import type { Options as WorkerOptions } from "./buildWorker";
 
 type Options = Omit<
 	WorkerOptions,
-	"fallbackService" | "buildOutputDirectory" | "nodejsCompat"
->;
+	"outfile" | "fallbackService" | "buildOutputDirectory" | "nodejsCompat"
```

Comment:
we don't seem to be using `outfile` anywhere. Do we still need here then?

Reply:
This is `Omit`ing ` the outfile property :)

Reply:
lol...ignore me 😅